src/main/java/uk/gov/justice/digital/delius/config/SwaggerConfig.java

```diff
@@ -37,6 +38,23 @@ public Docket communityAPI() {
         return new Docket(DocumentationType.SWAGGER_2)
                 .useDefaultResponseMessages(false)
                 .groupName("Community API")
+                .tags(new Tag("-- Popular core APIs --", "Endpoints that are most often used by clients"),
+                        new Tag("Convictions", "Offender conviction related, also known as Events in Delius"),
+                        new Tag("Contact and attendance", "Offender case notes, also known as Contacts in Delius and attendances"),
+                        new Tag("Custody", "Offender custody information"),
+                        new Tag("Offender managers", "Community and prison offender managers and responsible officers"),
+                        new Tag("Documents", "Documents related to a offender and their sentence"),
+                        new Tag("Sentence requirements and breach", "Offender sentence requires and breaches of those requirements"),
+                        new Tag("Personal circumstances", "Offender personal circumstances, e.g. medical condition"),
+                        new Tag("Registrations", "Offender registers, e.g sex offenders register, TACT, MAPPA etc"),
+                        new Tag("Authentication and users", "Authentication and authorisation endpoints for access roles and user details"),
+                        new Tag("Court appearances", "Offender court appearances that have been recorded by probation"),
+                        new Tag("Staff", "Staff related endpoints. Staff are typically offender managers and maybe users of Delius"),
+                        new Tag("Events", "Endpoints to support propagation of offender related events"),
+                        new Tag("Reference data", "Retrieves Delius reference data"),
+                        new Tag("Sentence dates", "Endpoints and set a retrieve sentence dates"),
```

Comment:
Does this read right to you or is it just me? `Endpoints and set a retrieve sentence dates` meh

Reply:
No, it is garbage grammar - I'll correct